### ISSUE TITLE
GetGamesList error

### ISSUE THREAD

**Issue by Touitoui** [TheGamesDB/TheGamesDB#141]
When searching a game, unrelated games will show up.
<b>Ex</b> : <a href="http://thegamesdb.net/api/GetGamesList.php?name=diablo">thegamesdb.net/api/GetGamesList.php?name=diablo</a> will show every <b>Diablo</b>, but <b>Deadpool</b> and <b>Deep blue</b> will be here too.
(The same thing happens with <b>Pokemon</b> which give <b>Pikmin</b> and <b>Pac-mac</b>)

**Comment by Solidus0815** [comment]
Seems like this is related to the phonectic search.
See line 26 in GetGamesList.php ( GameTitle SOUNDS LIKE '$name' )
Fix depends on what you want to archive.

**Comment by flexage** [comment]
This is true, a short while ago we implemented an elastic search server and moved the site over to it. It is producing much better results, however we never got around to moving the API over to use elastic search. If anybody wants to take a stab at it please feel free, there are some tools in our admin control (built into thegaesdb source code, visible when you sign into your dev site as an administrator) panel to properly populate your own elastic search data.
